.ipynb_checkpoints/MSDS Data Mining Project 1-checkpoint.ipynb

```diff
@@ -21,9 +21,16 @@
    "source": [
     "### Business Understanding:\n",
     "\n",
-    "This paper is an evaluation of the dataset used in the research article "Effects of earthquake on perinatal outcomes: A Chilean register-based study" by Yasna K. Palmeiro-Silva, et al (Plos One, https://doi.org/10.1371/journal.pone.0191340). The study reported whether there was an effect of the February 27, 2010 earthquake that occurred in Chile on the pregnancy outcomes of Chilean women who were pregnant at the time of the earthquake. The data set is available at     https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/F7XCI7&version=1.0. \n",
+    "This paper is an evaluation of the dataset used in the research article \"Effects of earthquake on perinatal outcomes: A Chilean register-based study\" \n",
+    "by Yasna K. Palmeiro-Silva, et al . The study addressed the question of whether there was an \n",
```

Comment:
##Add a statement that we needed to replace two missing values in Parity##
The data that was downloaded was in Spanish and was in a Stata file. A data summary in Stata indicated no missing values. The data were then exported from Stata as a .csv file. Re-evaluation of the data after this export revealed two NaN values in the variable Parity. Comparison with the Stata file indicated that both values should be zero, and the .csv data file was amended as such. These two values are also consistent with the median value of the Parity variable. We translated the column names and the category names of categorical variables into English, as described in the Data Preparation section.